### PR TITLE
Do all dependabot updates and fix issues

### PR DESCRIPTION
**Pull request by r-ash**
## Description

This PR does all currently open dependabot updates, and fixes up any issues introduced from them
* #938 
* #979 & https://github.com/hivtools/hint/pull/1003
* #986 

Note the end to end test updates, there were some small spacing changes in this update

## Type of version change

None

## Checklist

- [ ] I have incremented version number, or version needs no increment
- [ ] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/src/tests/e2e/reviewInput.spec.ts [hunk 1]
```diff
@@ -60,9 +60,8 @@ test("can view time series plot", async ({ projectPage }) => {
     // Note we need to wrap this with an expect.toPass. Fetching ANC data takes a while,
     // toHaveScreenshot checks the expectation as soon as 2 sequential screenshots return the same data. So because
     // ANC data fetch takes a while, this is taking 2 screenshots before the UI updates, returning and erroring.
-    await expect(async() => {
-        await expect(page.locator("#review-inputs")).toHaveScreenshot("time-series-anc.png")
-    }).toPass();
+    await page.waitForResponse(resp => resp.url().includes('/chart-data/input-time-series/anc') && resp.status() === 200)
+    await expect(page.locator("#review-inputs")).toHaveScreenshot("time-series-anc.png")
```
Comment (r-ash) [inline src/app/static/src/tests/e2e/reviewInput.spec.ts:64]: I think this will be a little less flaky version I hope!

**Review by M-Kusumgar — CHANGES_REQUESTED** [review]
this looks good, btw just spotted something weird in the table, which is not there in mrc-4856, in the table plot:
1. deselect both in sex filter
2. select both again

in this version its a bit weird, in the old version the table just filled the entire space, i think we mightve missed something in the table component

File: src/app/static/src/app/components/plots/table/TableDisplay.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@ const defaultColDef = {
     // suppressMenu hides the filter menu which showed on the column title
```
Comment (M-Kusumgar) [inline src/app/static/src/app/components/plots/table/TableDisplay.vue:29]: also just spotted this 
```suggestion
    // suppressHeaderMenuButton hides the filter menu which showed on the column title
```

**Review by M-Kusumgar — APPROVED** [review]
Niceeeee

**Pull request merged**